### PR TITLE
chore(Examples): Setup auto-deploy for Firebase.

### PR DESCRIPTION
**Pull request by matanlurey**
Closes https://github.com/dart-lang/angular/issues/939.

I manually pushed once: https://dart-hn.firebaseapp.com/.

(DO NOT MERGE)

/cc @filiph

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -149,6 +149,11 @@ stages:
 branches:
   only: [master]
 
+# MANUALLY ADDED. Do not remove.
```
Comment (leonsenft) [inline .travis.yml]: Does mono_repo not yet support adding `after_success` commands?

Comment (matanlurey) [reply .travis.yml]: No, it does not.

**Comment by leonsenft** [comment]
> I manually pushed once

By running the `./tool/deploy.sh`? Just curious if Firebase can just "figure it out" without being invoked from the project root (`examples/hacker_news_pwa/`).

**Comment by matanlurey** [comment]
@leonsenft No, you do need to "CD". I'll fix that.

File: .travis.yml [hunk 2]
```diff
@@ -149,6 +149,11 @@ stages:
...
+after_success:
```
Comment (alorenzen) [inline .travis.yml]: Will this run after every green travis run, even just for a pending PR that we have not actually approved yet?

Comment (matanlurey) [reply .travis.yml]: No, only after merged. The naming is meh, not my choice.

**Pull request closed without merging**